### ISSUE TITLE
does not scan `t()` declarations if translation key is in the new line, `v2.9.1`

### ISSUE THREAD

**Issue by mvirbicianskas** [i18next/i18next-scanner#131]
#### Version
- i18next: <i>3.1.0</i>
- i18next-scanner: <i>2.6.6</i>

#### Configuration
```js
{
            ns: ['international'],
            defaultNs: 'international',
            lngs: ['en-DE', 'de-DE'],
            resource: {
                loadPath: '<%= dirs.src %>locale/%(lng)s/%(ns)s.json',
                savePath: 'locale/%(lng)s/%(ns)s.json'
            },
            interpolation: {
                prefix: '%(',
                suffix: ')s'
            },
            nsSeparator: false,
            keySeparator: false,
            initImmediate: true
        }
```

Hii, everyone,

I'm not sure if this is a duplicate but I scanned through issues and did not find anything relevant, so here it is:

In my use case, we sometimes have very long translation keys, which could potentially be `prettyfied`.

So this -
```
const translation = i18n.t("Very long translation key which will be pre-formatted with prettier we use thus i18next scanner will not catch this translation")
```

Becomes this -
```
const translation = i18n.t(
    "Very long translation key which will be pre-formatted with prettier we use thus i18next scanner will 
          not catch this translation"
)
```

Basically meaning i18next-scanner somehow omits translation keys with the following format:
```
t(\n"Very long translation key which will be pre-formatted with prettier we use thus i18next scanner will not catch this translation"\n)
```

Does anyone know a solution to this problem?  Or a workaround?

**Comment by mvirbicianskas** [comment]
So I looked into source code, it seems that the problem originates from `parseFuncFromString` where regexp pattern is being created, and the regex for some reason does not match `i18n.t`-ions if they have white space at the closing `)` parentheses, just checked on regexr dot com with the example I've provided ☝️ 
Resulting RegExp pattern:
`(?:(?:^\s*)|[^a-zA-Z0-9_])(?:(?:i18n\.t))\(([\r\n\s]*`(?:[^`\\]|\\(?:.|$))*`|[\r\n\s]*"(?:[^"\\]|\\(?:.|$))*"|[\r\n\s]*'(?:[^'\\]|\\(?:.|$))*'[\r\n\s]*)(?:[\,]([\r\n\s]*`(?:[^`\\]|\\(?:.|$))*`|[\r\n\s]*"(?:[^"\\]|\\(?:.|$))*"|[\r\n\s]*'(?:[^'\\]|\\(?:.|$))*'[\r\n\s]*))?[\,\)]`
Does not match
```
i18n.t(
        "It seems you're trying to login with an email which does not have a password. Click reset password link to send an email with instructions to set your new password."
    )
```
But matches
```
i18n.t(
        "It seems you're trying to login with an email which does not have a password. Click reset password link to send an email with instructions to set your new password.")
```
 
:(((

**Comment by mvirbicianskas** [comment]
Lastly for someone stumbling upon this version `2.6.6` does scan this type of format.
https://github.com/i18next/i18next-scanner/pull/113/files#diff-6947033678b93d106e25614dd972e66fR346 - this was the PR that introduced such a change, unfortunately :( 
I'm not an expert in RegExp though, and it's a bit overwhelming for me to point out what's the exact issue with this RegExp, maybe @daliusd could provide any insight here? Many thanks 🙏

**Comment by daliusd** [comment]
There is pull request for this https://github.com/i18next/i18next-scanner/pull/123

**Comment by mvirbicianskas** [comment]
awesome ;) but I guess it'll take some time to merge it :D

**Comment by cheton** [comment]
Published v2.9.2. Thank you!

**Comment by mvirbicianskas** [comment]
nice! thank you very much 🙏
